go-controller/pkg/crd/userdefinednetwork/v1/shared.go

```diff
@@ -41,7 +44,7 @@ type Layer3Config struct {
 	//
 	// MTU is optional, if not provided, the globally configured value in OVN-Kubernetes (defaults to 1400) is used for the network.
 	//
-	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Minimum=576
```

Comment:
minimum mtu for v4 is 576 and v6 is 1280?

Reply:
Yes, according to the referenced RFCs.

These are also the minmial MTU CNO checks https://github.com/openshift/cluster-network-operator/blob/master/pkg/network/ovn_kubernetes.go#L908
https://github.com/openshift/cluster-network-operator/blob/master/pkg/network/mtu.go#L13